Makefile.am

```diff
@@ -53,10 +55,14 @@ install-data-hook:
 	$(MKDIR_P) '$(DESTDIR)$(systemdunitdir)'
 	$(INSTALL_DATA) $(srcdir)/var-endless_x2dextra.mount \
 	  '$(DESTDIR)$(systemdunitdir)/var-endless\x2dextra.mount'
+	$(MKDIR_P) '$(DESTDIR)$(systemdunitdir)/eos-factory-reset.target.wants'
+	ln -s '../eos-factory-reset.service' \
+	  '$(DESTDIR)$(systemdunitdir)/eos-factory-reset.target.wants'
```

Comment:
Do other packages which define new targets do this? I believe that (the equivalent of) `systemd preset-all` is run at boot (on unconverted systems) which enables any unit not explicitly disabled in `50-eos.preset` (in this repo). eos-factory-helper, for example, defines a new .target and .service but doesn't do anything special beyond installing the files.